configure.sh

```diff
@@ -87,10 +87,8 @@ export TF_VERSION="${TF_VERSION:-1.15.0}"
 export TF_VERSION_UNDERSCORE=$(echo $TF_VERSION | sed 's/\./_/g')
 export TF_VERSION_DASH=$(echo $TF_VERSION | sed 's/\./-/g')
 
-cat WORKSPACE | sed "s/TF_VERSION/${TF_VERSION_UNDERSCORE}/" > .WORKSPACE.tmp
-mv .WORKSPACE.tmp WORKSPACE
-cat pip_pkg_scripts/setup.py | sed "s/TF_VERSION/${TF_VERSION_DASH}/" > pip_pkg_scripts/.setup.py.tmp
-mv pip_pkg_scripts/.setup.py.tmp pip_pkg_scripts/setup.py
+cat WORKSPACE.in | sed "s/TF_VERSION/${TF_VERSION_UNDERSCORE}/" > WORKSPACE
```

Comment:
where is this 'WORKSPACE.in'? did you forget to copy one from WORKSPACE?

Reply:
WORKSPACE.in is the WORKSPACE Template. Right I just `mv WORKSPACE WORKSPACE.in`
Imagine such situation:
You use tensorflow-1.15.0 currently. After you ./configure.sh, you got WORKSPACE with TF_VERSION replaced by the 1-15-0
Then, you upgrade tensorflow from 1.15.0 to 2.0.0. And when you ./configure.sh, there is no TF_VERSION in WORKSPACE now. And the TF_VERSION part in your WORKSPACE is still 1-15-0 instead of 2-0-0.

This PR solves the situation described above.

Reply:
That makes sense. I did not see the rename.